### PR TITLE
Pizzeria Penguin

### PR DESCRIPTION
**Pull request by linaroos**
## Netlify link
Add your Netlify link here.

### CODE REVIEW

**Comment by AntonellaMorittu** [comment]
Hi Lina, please edit the pull request message to add a netlify link. If you need, check how to do it [here](https://www.notion.so/technigo/Zoom-recordings-852e9d63af0b40a6b9a63f21313c0d6c?pvs=4#eacbbd3715f646dda266a56c0a7fd722) Thank you 🙌

**Comment by linaroos** [comment]
Hi Antonella, I think I really got the logic of the PR and Netlify now :-) Grazie Mille!

File: .DS_Store
Comment (AntonellaMorittu) [inline .DS_Store:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) 👍

File: code/script.js [hunk 1]
```diff
@@ -1,19 +1,62 @@
...
-
-// Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  "Welcome to Pizzeria Penguin. Ready to order? - Click 'OK' to begin.")
+
+//user name//
+const userName = prompt("Please enter your name:")
+console.log(userName)
```
Comment (AntonellaMorittu) [inline code/script.js:6]: Please remove the the console.log() around the code. Generally they need to be deleted when the code has been tested and works as meant to. 🧹

File: code/script.js [hunk 2]
```diff
@@ -1,19 +1,62 @@
...
+  alert("Invaild choice, please start again")
+}
+
+console.log("selected meal", selectedMeal)
+alert(`You chose ${selectedMeal} ,exellent choice!`)
 
-// Step 2 - Food choice
-// Your code goes here
+//user selection of age//
+let customerType = prompt ("Is the order for a child or an adult?")
```
Comment (AntonellaMorittu) [inline code/script.js:37]: be a little be more descriptive of what is the action of the user in this case, what should it type? Write an explicit instruction to improve the app usability 👍

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Lina, well done finishing your week 1 project! ⭐

Have you checked the link of your deployed version? The link I have found on your `README.md` is not perhaps reflecting your current code implementation? Please take a look :)

The Netlify link should be included in your pull request (check pull request template from the orginal repository). You should copy and paste the content and adapt it with your info (this week, the deployed version link).

You have almost completed all the basic requirements for this week, but right now the app is not working as it should.

Check [step 3](https://github.com/linaroos/project-pizza/blob/main/instructions.md#step-3---subtype-choice) of the instructions please 🙌 

Generally, it's good practice to remove every single file that's not useful for the application, like `pull_request_template.md`. That's there for you to copy the content and use it in the PR, after that can be deleted.

I left some comments around the code to help you understand where changes need to be applied. Take a look please, thank you 🙌

File: .DS_Store
Comment (linaroos) [reply .DS_Store:1]: Alright, I actually think I managed to fix this 🤗

File: code/script.js (hunk 1 above)
Comment (linaroos) [reply code/script.js]: Well noted and changes made, did not actually not know this.

File: code/script.js (hunk 2 above)
Comment (linaroos) [reply code/script.js]: Well noted and changed.

**Comment by linaroos** [comment]
I am really having huge problems with my Netlify :-( I have been sitting for hours and hours, checking google, chat gpt but the error is just to complicated. I would truly appreciate to get some help with this, had some problems last time to. Many thanks!

**Review by AntonellaMorittu — APPROVED** [review]

**Pull request closed without merging**